### DISCUSSION TITLE
minio s3 does not work

### DISCUSSION THREAD

**Discussion by wenqvip** [outline/outline#2466] [Self Hosting] [answered]
Hello, I use local minio server instead of aws s3 server. I have prepared it successfully, and can use mc to putObject.
then I set the .env file like this:
```
AWS_ACCESS_KEY_ID=wxxxxxp
AWS_SECRET_ACCESS_KEY=pxxxxxx0
AWS_REGION=cn-west-1
AWS_S3_UPLOAD_BUCKET_URL=http://localhost:9000
AWS_S3_UPLOAD_BUCKET_NAME=outline
AWS_S3_UPLOAD_MAX_SIZE=262144000
AWS_S3_FORCE_PATH_STYLE=true
AWS_S3_ACL=private
```
but can not set avatar in outline, everytime show error: "Fail to Fetch". And cant upload attachment pics too.
And I found the outline does not perform a connection to localhost:9000. (I use `netstat -anopt | grep 9000` to check that)

do someone have some advice ?

**Comment by wenqvip** [comment, marked as answer]
hello all,
I find the reason why outline does not connect to minio: the URL need to be https.
I guess the aws s3 sdk stop continuing when found the URL is not https.

**Reply by almereyda** [reply]
Can you mark your answer as an accepted answer, or does this need to be done by the moderators/maintainers?
